Hyper/Window.cs

```diff
@@ -192,7 +192,7 @@ protected override void OnFocusedChanged(FocusedChangedEventArgs e)
 
     private bool IsGameReady()
     {
-        return _sinceGameStartStopwatch.ElapsedMilliseconds > 1000;
+        return true;
```

Comment:
why change this?

Reply:
Visual studio auto merged this quite poorly. I'll change this